### ISSUE TITLE
Is the file of class SendPushNotificationToApnsTask located in the apns module or the request module?

### ISSUE THREAD

**Issue by MobWiMetro** [ZsoltSafrany/java-apns-gae#2]
Hi,

Thanks for sharing the code. I just want to know which module I should put the class file SendPushNotificationToApnsTask.

**Comment by ZsoltSafrany** [comment]
Hey, good question.

It makes sense to ask questions like "If I have more than one module then should I separate my code into these modules in a way that reflects the logical difference between these modules?".

I remember I wanted to put all my APNS related source and resource files into my `apns` module. But I had some technical problems with that (e.g. the classes in my not-main module were not found during runtime) so eventually I put all my java code into my main module. I don't remember the exact problem and I can't tell for sure whether it was an App Engine shortcoming or I just could figure out the way to do it.

Bottom line, in my case modules (except the main one) contained only descriptor files (and no source or resource files).
